src/Components/Skills/Skills.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+function Skills() {
+  return (
+    <div>
+      <section id="Skills" className='bg-black bg-opacity-30 shadow-lg rounded-2xl p-5 text-orange-500 '>
```

Comment:
**Imprve this background color.**

Reply:
Hi  Himalf, I have implemented the changes as per your suggestions:

Updated the color scheme to a more neutral color.
Added spacing between the icons and text in the <li> elements.
Could you please review the updates and let me know if further adjustments are needed? Thanks!"

Reply:
**Ok nimesh631 I have reviewed your changes**